examples/datasets_tutorial.py

```diff
@@ -55,16 +55,28 @@
 ############################################################################
 # Get the actual data.
 #
-# Returned as numpy array, with meta-info
-# (e.g. target feature, feature names, ...)
-X, y, attribute_names = dataset.get_data(
+# The dataset can be returned in 2 possible formats: as a NumPy array, a SciPy
+# sparse matrix, or as a Pandas DataFrame (or SparseDataFrame). The format is
+# controlled with the parameter ``dataset_format`` which can be either 'array'
+# (default) or 'dataframe'. Let's first build our dataset from a NumPy array
+# and manually create a dataframe.
+X, y, attribute_names = dataset.
```

Comment:
I believe this is the line where the examples fail due to a syntax error, right?